tools/debian/rules

```diff
@@ -2,6 +2,9 @@
 
 DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 
+# we need an apparmor profile for >= 4.0; this exists in Ubuntu >= 24.04, but not in Debian yet
+PRE4AA = $(filter $(shell . /etc/os-release; echo $${VERSION_ID:-unstable}),22.04 11 12 unstable)
```

Comment:
This seems awfully brittle...

Why can't we query the installed package version instead?  Is it not available during the build?  Could we add it as a build dependency?

Reply:
We could add it as a build dependency too, that's rather heavy-handed though. I.e. we'd need to add it to our four images

Reply:
naja...